modules/Sources/Features/Settings/SettingsStore.swift

```diff
@@ -24,61 +18,46 @@ public struct SettingsReducer: Reducer {
     public struct State: Equatable {
         public enum Destination {
             case about
-            case backupPhrase
-            case privateDataConsent
-            case serverSetup
+            case advanced
         }
 
+        public var advancedSettingsState: AdvancedSettingsReducer.State
         @PresentationState public var alert: AlertState<Action>?
         public var appVersion = ""
         public var appBuild = ""
         public var destination: Destination?
         public var isRestoringWallet = false
-        public var phraseDisplayState: RecoveryPhraseDisplayReducer.State
-        public var privateDataConsentState: PrivateDataConsentReducer.State
-        public var serverSetupState: ServerSetup.State
         public var supportData: SupportData?
         
         public init(
+            advancedSettingsState: AdvancedSettingsReducer.State,
             appVersion: String = "",
             appBuild: String = "",
             destination: Destination? = nil,
             isRestoringWallet: Bool = false,
-            phraseDisplayState: RecoveryPhraseDisplayReducer.State,
-            privateDataConsentState: PrivateDataConsentReducer.State,
-            serverSetupState: ServerSetup.State,
             supportData: SupportData? = nil
         ) {
+            self.advancedSettingsState = advancedSettingsState
             self.appVersion = appVersion
             self.appBuild = appBuild
             self.destination = destination
             self.isRestoringWallet = isRestoringWallet
-            self.phraseDisplayState = phraseDisplayState
-            self.privateDataConsentState = privateDataConsentState
-            self.serverSetupState = serverSetupState
             self.supportData = supportData
         }
     }
 
     public enum Action: Equatable {
+        case advancedSettings(AdvancedSettingsReducer.Action)
         case alert(PresentationAction<Action>)
-        case backupWalletAccessRequest
         case onAppear
-        case phraseDisplay(RecoveryPhraseDisplayReducer.Action)
-        case privateDataConsent(PrivateDataConsentReducer.Action)
         case restoreWalletTask
         case restoreWalletValue(Bool)
```

Comment:
Are these still needed here? I see they were added to `AdvancedSettingsStore`m like other enum variants that were removed from here.

Reply:
Both are used for a different thing unrelated to this ticket. It's about this restore UI. The actions are used to subscribe to the service that updates the state. The UI is rendered on all possible screens so both Settings and AdvancedSettings present it.

<img width="616" alt="Screenshot 2024-02-14 at 11 13 56" src="https://github.com/Electric-Coin-Company/zashi-ios/assets/7198042/8b65c42d-b834-4ba9-8ca6-12e878646623">